redaxo/src/core/lib/setup/setup.php

```diff
@@ -257,6 +257,11 @@ public static function isInitialSetup(): bool
             return 0 == $userSql->getRows();
         } catch (rex_sql_could_not_connect_exception $e) {
             return true;
+        } catch (rex_sql_exception $e) {
+            if ($e->getSql() && rex_sql::ERRNO_TABLE_OR_VIEW_DOESNT_EXIST !== $e->getSql()->getErrno()) {
```

Comment:
Muss es hier nicht eher so sein?

```suggestion
            if (!$e->getSql() || rex_sql::ERRNO_TABLE_OR_VIEW_DOESNT_EXIST !== $e->getSql()->getErrno()) {
```